tests/Operation/ModifyCollectionFunctionalTest.php

```diff
@@ -29,9 +33,23 @@ public function testCollMod(): void
             ['index' => ['keyPattern' => ['lastAccess' => 1], 'expireAfterSeconds' => 1000]],
             ['typeMap' => ['root' => 'array', 'document' => 'array']],
         );
-        $result = $modifyCollection->execute($this->getPrimaryServer());
+        $modifyCollection->execute($this->getPrimaryServer());
```

Comment:
I'm not sure that the return value of `collMod` is entirely irrelevant, as the [server docs include a note](https://www.mongodb.com/docs/manual/reference/command/collMod/#hide-an-index-from-the-query-planner) about being able to infer whether a value actually changed server-side by inspecting the command result:

> If the operation is successful but the hidden value has not changed (specifically, hiding an already hidden index or unhiding an already unhidden index), the command omits the `hidden_old` and `hidden_new` fields from the output.

I didn't consider that until seeing this test change.

If you agree, I'd suggest just reverting the changes to ModifyCollection. Otherwise, feel free to leave as-is.

Reply:
I reverted the change on this operation.